### ISSUE TITLE
Login or enter credentials before starting the audit

### ISSUE THREAD

**Issue by niraj8** [mfrachet/cypress-audit#6]
Currently, I don't think there is a way for now, to interact with the page before it starts the `audit`.

How can we add this functionality to cypress-audit?

**Comment by mfrachet** [comment]
You can run cypress commands before the audit. 

Can you provide a gif or a video or even a repro of what you're attempting to do? So that I can play with it and check what I can do? 😄

**Comment by niraj8** [comment]
I am trying to add the audit task in the e2e test suite for an application I am working on. 

```
it('should verify the lighthouse scores', function() {
  cy.adminLogin();
  cy.audit();
});
```
The application has a login page and I have set up the `adminLogin` task to fill in the credentials. 

The `audit` task opens up a new window and audits just the login screen. As the bulk of the application is loaded after a successful login, this doesn't really audit the application.

Am I missing something?😅

**Comment by mfrachet** [comment]
Nope you're not missing anything 😛. Is the application open source? So that I can play with it and make some experimentations? :)

**Comment by mfrachet** [comment]
Okay, I may have an idea. Can you try running something like:

```javascript
cy.audit({ performance: 10 }, { disableStorageReset: true });
```

and tell me if it works better?

**Comment by mfrachet** [comment]
@niraj8 thanks to your command I was able to release a new version of the module that is aware of cookies in lighthouse.

I've also pushed an example based on a (fake) login phase that relies on cookies: https://github.com/mfrachet/cypress-audit/blob/master/example/cypress/integration/examples/homepage.spec.js#L3

I've also tried to run it in a real world application I'm using and it works on my side :). 

Feel free to install the v.0.0.5 version and give feedback!

Thanks again for this, you rock 🎸 🚀
